SimpleBrowser.WebDriver/SimpleBrowser.WebDriver.csproj

```diff
@@ -33,13 +33,14 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Moq, Version=4.2.1510.2205, Culture=neutral, PublicKeyToken=69f491c39445e920, processorArchitecture=MSIL">

```

Comment:
This seems strange: why would the library need a dependency on Moq. That should only be used in the Test project.

Reply:
No idea, it was there before, I've just updated the version.

Reply:
No, it wasn't there. Must have happened as part of the update step. No problem. I'll take it out and try to run a CI build on the branch.